tests/AppBundle/Controller/Legislatives/HomeControllerTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Tests\AppBundle\Controller\Legislatives;
+
+use AppBundle\DataFixtures\ORM\LoadLegislativesData;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+use Tests\AppBundle\SqliteWebTestCase;
+
+/**
+ * @functional
+ */
+class HomeControllerTest extends SqliteWebTestCase
+{
+    use ControllerTestTrait;
+
+    public function testLegislativesCandidatesDirectory()
+    {
+        $crawler = $this->client->request(Request::METHOD_GET, 'https://legislatives-en-marche.dev/');
```

Comment:
Maybe we should store this url in a constant and use it everywhere needed ? (also update the config for use the constant)